src/helm/benchmark/presentation/run_specs_cleva_v1.conf

```diff
@@ -0,0 +1,298 @@
+# CLEVA (v1) RunSpecs (https://arxiv.org/pdf/2308.04813.pdf)
+
+entries: [
+    ######################################################### Application ######################################################
+
+    {description: "cleva:model=full_functionality_text,task=dialogue_generation,subtask=task_oriented,prompt_id=0,version=v1,data_augmentation=cleva", priority: 1}
```

Comment:
For runs that use the `ADAPT_MULTIPLE_CHOICE_JOINT` or `ADAPT_GENERATION` adapters, could you change `model=full_functionality_text` to `model=text`?

The reason for this is that some models like GPT-3.5 and GPT-4 are _not_ `full_functionality_text` models because they don't return logprobs, but we still want to be able to run them on a subset of these scenarios that don't need logprobs.